glide.lock

```diff
@@ -129,7 +130,7 @@ imports:
   subpackages:
   - prometheus
 - name: github.com/prometheus/client_model
-  version: fa8ad6fec33561be4280a8f0514318c79d7f6cb6
+  version: 6f3806018612930941127f2a7c6c453ba2c527d2
```

Comment:
We usually prefer deterministic behavior by pinning the package versions so it is guaranteed that glide update will always install the same versions. As a side benefit, it also reduces noises when someone run glide update next time.